src/aspire/image/__init__.py

```diff
@@ -117,8 +117,10 @@ def copy(self):
     def shift(self, shifts):
         """
         Translate image by shifts. This method returns a new Image.
+
```

Comment:
Why is this file being modified in this PR?

Reply:
Probably  change accidentally.

Reply:
if this is going to conflict with your other PR we should remove it, otherwise fbm thanks

Reply:
> Probably change accidentally.

Ok. Please restore the file in that case.

Reply:
Actually the empty line was added to generate a space between the first title line and the rest so that it becames consistent with other functions in the same class. There is no conflict with others.

Reply:
Yes but it doesn't belong here. We can leave it this time, but in the future don't bring in accidental changes into a PR. It makes them noisier to review and the commit history doesn't make sense. (Looks like these changes occurred in 25f92b7f4f5a6fa30edc9c4848f5c4dd35a4130c, which has nothing to do with fixing docstrings.) If you want to fix up the docstrings or clean up whitespace, that can be in a separate PR.

Reply:
Got. I will try to avoid touching unrelated items. This probably happened when I tried to implement the phase_flip function over there but we reimplemented it as `LambdaFilter` later.